core/src/ast/function.rs

```diff
@@ -335,6 +336,19 @@ impl ToSql for Function {
                 format!("UNWRAP({}, {})", expr.to_sql(), selector.to_sql())
             }
             Function::GenerateUuid() => "GENERATE_UUID()".to_owned(),
+            Function::Greatest(items) => {
+                let items = items
+                    .iter()
+                    .filter(|&item| !matches!(item, Expr::IsNull(_)))
+                    .map(ToSql::to_sql)
+                    .collect::<Vec<_>>()
+                    .join(", ");
+                if items.is_empty() {
+                    "NULL".to_owned()
```

Comment:
It would be nice to have a test that returns NULL for empty items.

Reply:
may be this causes some confusion..?

though `is_empty` it does not mean that the result of `to_sql` is `NULL`.
we can simply use `GREATEST()` for that.